apps/docs/docs/user/core-concepts.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 2
+---
+
+# Core Concepts
+
+The core concepts of Jayvee are `Pipelines`, `Blocks`, and `ValueTypes`.
+
+## Pipelines
+
+A `Pipeline` is a sequence of different computing steps, the `Blocks`. The default output of a block becomes the default input of the next block, building a chain of computing steps. In the scope of a `Pipeline`, you can connect these blocks via the `pipe` syntax:
+
+```javascript
+pipeline CarsPipeline {
+    // Assumption: blocks "GasReserveHttpExtractor", "GasReserveCSVInterpreter", "GasReserveTableInterpreter", and "GasReserveLoader" are defined
+
+    GasReserveHttpExtractor
+		-> GasReserveTextFileInterpreter
+		-> GasReserveCSVInterpreter
+		-> GasReserveTableInterpreter
+		-> GasReserveLoader;
+}
+```
+
+Alternatively, you can use a slightly longer syntax for pipes:
+
+```javascript
+pipeline CarsPipeline {
+    // Assumption: blocks "GasReserveHttpExtractor", "GasReserveCSVInterpreter", "GasReserveTableInterpreter", and "GasReserveLoader" are defined
+
+    pipe {
+        from: GasReserveHttpExtractor;
+        to: GasReserveTextFileInterpreter;
+
+    }
+
+    pipe {
+        from: GasReserveTextFileInterpreter;
+        to: GasReserveCSVInterpreter;
+
+    }
+
+    // etc.
+}
+```
+
+
+## Blocks
+
+A `Block` is a processing step within a `Pipeline`. It can have a default input and a default output. We differentiate the following types of `Blocks`:
+- `ExtractorBlocks` do not have a default input but only a default output. They model a **data source**.
+- `TransformatorBlocks` have a default input and a default output. They model a **transformation**.
+- `LoaderBlocks` do have a default input but nor a default output. They model a **data sink**.
+
+The general structure of a `Pipeline` consisting of different blocks is the following:
+```mermaid
+flowchart LR
+    A[ExtractorBlock] --> B(TransformatorBlock)
+    B --> C(TransformatorBlock)
+    C --> D(LoaderBlock)
+```
+
+The common syntax of blocks is at its core a key-value map to provide configuration to the block. The availability of attribute keys and their respective `ValueTypes` is determined by the type of the `Block` - indicated by the identifier after the keyword `oftype`:
```

Comment:
Felix renamed attributes to properties so renaming all references to attributes to properties would make sense.

Reply:
Addressed in [b809763](https://github.com/jvalue/jayvee/pull/244/commits/b809763ac8ea5296b286f6ac862ab440e604632b)